packages/runtime/lib/runtime.js

```diff
@@ -210,6 +229,7 @@ class Runtime extends EventEmitter {
     this.#updateStatus('closing')
 
     clearInterval(this.#metricsTimeout)
+    process.removeListener('SIGUSR2', this.restartSignalListener)
```

Comment:
Why is this needed here? The use of `Runtime` should not add any signal to process. These are handled in `start`: https://github.com/platformatic/platformatic/blob/0bed5becb8d726dc8932c1876bec793001a31ee1/packages/runtime/lib/start.js#L38.

Reply:
I moved into the runtime otherwise if a process received a SIGUSR2 after the runtime was closed it would have failed.